src/main/java/com/byznass/tiolktrack/postgres/ConnectionProvider.java

```diff
@@ -0,0 +1,8 @@
+package com.byznass.tiolktrack.postgres;
+
+import java.sql.Connection;
+
+public interface ConnectionProvider {
+
+	Connection getConnection() throws ConnectionFailureException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.byznass.tiolktrack.postgres.ConnectionFailureException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.tiolktrack.byznass.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)